schematics/transforms.py

```diff
@@ -426,54 +424,48 @@ def blacklist(*field_list):
 ###
 
 
-def convert(cls, instance_or_dict, trusted_data=None, partial=True, strict=False,
-            mapping=None, app_data=None, context=None):
-    field_converter = lambda field, value, context: field.to_native(value, context)
-    data = import_loop(cls, instance_or_dict, field_converter, trusted_data=trusted_data,
-                       partial=partial, strict=strict, mapping=mapping, app_data=app_data, context=context)
-    return data
+class FieldConverter(object):
 
+    def __call__(self, field, value, context):
+        raise NotImplementedError
 
-def to_native(cls, instance_or_dict, role=None, raise_error_on_role=True, app_data=None, context=None):
-    field_converter = lambda field, value, context: field.to_native(value, context)
-    data = export_loop(cls, instance_or_dict, field_converter, role=role,
-                       raise_error_on_role=raise_error_on_role, app_data=app_data, context=context)
-    return data
 
+class ExportConverter(FieldConverter):
```

Comment:
I can definitely agree with simplifying the built-in converters. I suppose it could make the program flow easier to understand. We can have a separate recipe to illustrate an advanced use case.